data/theme/gnome-shell-sass/_endless.scss

```diff
@@ -920,38 +920,69 @@ popup-separator-menu-item {
 }
 
 // Clubhouse Notifications View
+
+$clubhouse-notification-width: 444px;
+$clubhouse-image-width: 175px;
+$clubhouse-image-height: 177px;
+
 .clubhouse-notification {
-  font-size: 14pt;
-  width: 20em;
-  margin: 5px;
-  border-radius: 12px;
-  color: #000000;
-  background-color: #ffffff;
-  border: 1px solid #afafaf;
-  &:hover, &:focus { background-color: #dfdfdf; color: #000000; }
-
-  .notification-icon { padding: 5px; icon-size: 4.09em; }
-  .notification-content { padding: 5px; spacing: 5px; }
-  .secondary-icon { icon-size: 2.09em; }
-  .notification-actions {
+  background-color: transparent;
+  width: $clubhouse-notification-width;
+  &:hover, &:focus {
     background-color: transparent;
-    padding-top: 2px;
-    spacing: 1px;
   }
-  .notification-button {
-    padding: 4px 4px 5px;
-    background-color: #ffffff;
-    border: 1px solid #afafaf;
-    border-radius: 15px 15px;
+  .clubhouse-notification-close-button {
+    color: black;
+    padding: 7px 5px 0 0;
+  }
+  .notification-actions {
+    padding: 0;
+    margin: 0 0 0 30px;
+  }
+  .notification-button, .notification-button-next {
+    color: white;
+    background-color: #E97C28;
+    // @todo: This gradient background is what we want, but it breaks the box-shadow:
+    // background-gradient-start: #E97C28;
+    // background-gradient-end: #CE5611;
+    // background-gradient-direction: horizontal;
```

Comment:
Let's not add this comment. If we need to have it somewhere, please add it to the Phab task.

Reply:
Removed.